src/maskAnalysis/utils/monotoneChainConvexHull.ts

```diff
@@ -10,13 +15,13 @@ interface MCCHOptions {
  *
  * @param points - An array of points.
  * @param options - MCCH Algorithm options.
- * @returns Coordinates of the convex hull in clockwise order
+ * @returns Coordinates of the convex hull in clockwise order.
  */
 export function monotoneChainConvexHull(
   points: Point[],
-  options: MCCHOptions = {},
+  options: McchOptions = {},
 ): Point[] {
-  const { sorted } = options;
+  const { sorted = false } = options;
   if (!sorted) {
     points.sort(byXThenY);
```

Comment:
```suggestion
    points = points.slice().sort(byXThenY);
```

I know it's not new behavior, but let's fix it (sort is in-place).